### PR TITLE
contrib/clickclack: new package (0.2.3)

### PR DESCRIPTION
**Pull request by Froggo8311**

### CODE REVIEW

File: contrib/clickclack/update.py [hunk 1]
```diff
@@ -0,0 +1 @@
+pattern = "(?!archive\\/)[0-9\\.]*(\\.tar\\.gz)"
```
Comment (nekopsykose) [inline contrib/clickclack/update.py:1]: this returns a version that's just .tar.gz:  

```
Checking found version: .tar.gz
```

Comment (Froggo8311) [reply contrib/clickclack/update.py:1]: tf?? that shouldnt happen lol

Comment (Froggo8311) [reply contrib/clickclack/update.py:1]: OH

**Comment by Froggo8311** [comment]
oops, my dumbass accidentally used a negative lookahead group instead of a non-capturing group

**Comment by nekopsykose** [comment]
you need a group for the version part, you've done the opposite of how it works

File: contrib/clickclack/update.py [hunk 2]
```diff
@@ -0,0 +1 @@
+pattern = "\\/archive\\/([\\d\\.]+)\\.tar\\.gz"
```
Comment (nekopsykose) [inline contrib/clickclack/update.py:1]:
```suggestion
pattern = r"/archive/([\d\.]+)\.tar\.gz"
```

Comment (nekopsykose) [reply contrib/clickclack/update.py:1]: ah, the page doesn't even have version urls at all

**Pull request merged**